backend/src/ee/services/dynamic-secret/dynamic-secret-service.ts

```diff
@@ -1,4 +1,5 @@
 import { ForbiddenError, subject } from "@casl/ability";
+import { FastifyRequest } from "fastify";
```

Comment:
Avoid sharing server respective things service. The whole concept of service layer is it avoid making anything server specific outside. 

Please export this as a proper type if needed

Reply:
exported new type and replaced all references to this pattern